src/Miningcore/Blockchain/Ethereum/EthereumPayoutHandler.cs

```diff
@@ -979,6 +979,26 @@ private async Task PayoutBatchAsync(Balance[] balances)
             logger.Info(() => $"[{LogCategory}] Payouts complete.  Successfully processed top {txHashes.Count} of {balances.Length} payouts.");
         }
 
+        private async Task<decimal> GetMinimumPayout()
+        {
+            // The default minimum payout is based on the configuration
+            var minimumPayout = poolConfig.PaymentProcessing.MinimumPayment;
+
+            // If the GasDeductionPercentage is set, use a ratio instead
+            if (extraConfig.GasDeductionPercentage > 0)
+            {
+                var latestGasFee = await GetLatestGasFee() / EthereumConstants.Wei;
```

Comment:
CurrentGasFee already has the latest gas fee. Why do we need to call parity again?

Reply:
I assume that while we pay out our users, the gas fees are changing. Notice that the old payout method also called parity to get the gas fee for each balance it paid.